src/constants/plugins/GuiPlugins.ts

```diff
@@ -87,6 +95,43 @@ export const guiPlugins: { [pluginId: string]: GuiPlugin } = {
     nativePlugin: amountQuoteFiatPlugin,
     displayName: 'ACH Bank Transfer'
   },
+  interac: {
+    pluginId: 'amountquote',
+    storeId: '',
+    baseUri: '',
+    lockUriPath: true,
+    nativePlugin: amountQuoteFiatPlugin,
+    forceFiatCurrencyCode: 'iso:CAD',
+    displayName: ''
+  },
+  payid: {
+    pluginId: 'amountquote',
+    storeId: '',
+    baseUri: '',
+    lockUriPath: true,
+    nativePlugin: amountQuoteFiatPlugin,
+    forceFiatCurrencyCode: 'iso:AUD',
+    displayName: ''
+  },
+  pix: {
+    pluginId: 'amountquote',
+    storeId: '',
+    baseUri: '',
+    lockUriPath: true,
+    nativePlugin: amountQuoteFiatPlugin,
+    forceFiatCurrencyCode: 'iso:BRL',
+    displayName: ''
+  },
+  turkishbank: {
+    pluginId: 'amountquote',
+    storeId: '',
+    baseUri: '',
+    lockUriPath: true,
+    nativePlugin: amountQuoteFiatPlugin,
+    forceFiatCurrencyCode: 'iso:TRY',
+    defaultFiatAmount: '2000',
```

Comment:
Why is this amount so much higher than our normal 500?

Reply:
Turkish Lira is worth a LOT less

Reply:
500 Lira doesn't meet the minimum purchase

Reply:
Maybe we can do a TODO to have a conversion mechanism for fiats